api/schemas/flow-video.json

```diff
@@ -94,7 +94,7 @@
                             ]
                         },
                         "aspect_ratio": {
-                            "description": "The display aspect ratio.",
+                            "description": "The display aspect ratio. i.e. pixel_width / pixel_height",
```

Comment:
I don't think that change is valid for SD which has the complication of active areas etc. and pixel dimensions can vary when display aspect ratio should be set to 16:9 or 4:3 for example.

Reply:
Sorted in fixup